app/views/layouts/application.html.erb

```diff
@@ -4,12 +4,45 @@
     <title>Notation</title>
     <%= csrf_meta_tags %>
     <%= csp_meta_tag %>
-
+    <link rel="stylesheet" href="https://maxcdn.bootstrapcdn.com/bootstrap/3.4.1/css/bootstrap.min.css">
```

Comment:
question: Do you understand what the `.min` in the file name refers to?

Reply:
I didn't notice it then. Now I searched it and found that min.css excludes whitespace and indentation which make the file size decrease as compared to the plain CSS file.

Reply:
yes, there's a name for it and that is abbreviated as `min` in the filename. Do you know what `min` stands for?